app/services/album.js

```diff
@@ -0,0 +1,13 @@
+const request = require('../helpers/request'),
+  { apiBaseUrl } = require('../constants');
+
+const buildDefaultApiConfig = path => ({
+  url: `${apiBaseUrl}/${path}`,
+  json: true,
+  headers: { 'content-type': 'application/json' }
+});
+
+exports.getAlbums = () => request(buildDefaultApiConfig('albums'), error => new Error(error));
```

Comment:
declare errors inside the `errors` file rather than using `new Error` every time